src/java.base/share/classes/sun/nio/ch/Poller.java

```diff
@@ -274,18 +274,18 @@ private void subPollerLoop(Poller masterPoller) {
         }
     }
 
+    @Override
+    public String toString() {
+        return Objects.toIdentityString(this) + " [registered = " + map.size() + "]";
+    }
```

Comment:
Why did you move this and "inline" the content of `registered()`?

Reply:
I think that is PR merge issue, it's correct in the loom repo.